tests/syscall/mod.rs

```diff
@@ -137,11 +137,15 @@ fn get_att() {
     run_test(bin, 0, None, None, None);
 }
 
-#[cfg(feature = "backend-sgx")]
 #[cfg_attr(any(not(host_can_test_sgx), not(host_can_test_attestation)), ignore)]
 #[test]
 #[serial]
 fn sgx_get_att_quote() {
+    if let Ok(backend) = std::env::var("ENARX_BACKEND") {
```

Comment:
Is this env var documented anywhere? Is it only used for testing?

Reply:
this env var is one of the official ways to set the backend and the only way for tests to do it